tests/pytest/test_with_pytest.py

```diff
@@ -15,9 +15,9 @@ def test_simple(requests_mock):
 
 
 def test_redirect_and_nesting():
-    url_inner = "inner-mock://example.test/"
```

Comment:
I'm not sure these unit test changes were related, exactly. Maybe it's ok to not revert these?

Reply:
it was because `-` isn't a valid character in protocol and it was not actually being urlparsed. Just the test never relied on this.